NSS-Day-Cohort-58/rare-rest-git-it-girlz
# Ns comments

# Description

Added more function to the list view for comments. The server can now determine if a logged in user has created a comment or not based on a new virtual field 'created' that was added as a property decorator on the comment model. 


Please delete options that are not relevant.




# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration


**To test this branch:
git fetch--all
cd ns-comments

in browser, make sure that you can view a list of comments for posts and if the logged in user is the author of the comment, a button to delete the comment should appear. 

# Checklist:

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings
- [ x] New and existing unit tests pass locally with my changes
